src/main/java/no/nav/finnkandidatapi/permittert/PermittertArbeidssokerService.java

```diff
@@ -39,6 +33,11 @@ public PermittertArbeidssokerService(PermittertArbeidssokerRepository repository
     }
 
     public void behandleArbeidssokerRegistrert(ArbeidssokerRegistrertDTO arbeidssokerRegistrertDTO) {
+        log.info("Har mottatt en melding fra veilarbregistrering om bruker {} som {} registrerte status {} ",
+                "(anonymisert)",
+                arbeidssokerRegistrertDTO.getRegistreringTidspunkt(),
```

Comment:
Hva er hensikten med loggingen - er det en midlertidig logging vi vil ha i prod for å verifisere at alt går bra i begynnelsen, eller skal den være permanent? Blir det ikke i så fall veldig mange logginnslag?

Hvis innslaget skal være permanent tror jeg det kan være fornuftig å ikke logge registreringstidspunktet sammen med statusen. For å minske muligheten til å sammenstille informasjon fra forskjellige systemer. Eventuelt droppe statusen _og_ tidspunktet, og bare logge at vi har mottatt en melding

Reply:
Vel, om man vil ha den der permanent bør kanskje andre enn meg svare på.. Jeg bruker den nå for å verifisere at alt går bra, kan sikkert fjernes etter hvert.